src/test/ignore_nested.c

```diff
@@ -5,11 +5,17 @@
 // environment. The runner uses this to test
 // --nested=ignore in the absence of RR_UNDER_RR
 // environment variable.
-int main(int argc, char *argv[]) {
+int main(int argc, char *argv[], char *envp[]) {
```

Comment:
Ditto above re comment.

Reply:
This whole change to the test seems not needed in a tree that contains your vdso soname patch, so I remove this from my draft.